sympy/physics/mechanics/joint.py

```diff
@@ -490,13 +490,13 @@ def _fill_coordinate_list(self, coordinates, n_coords, label='q', offset=0):
             Count offset when creating new dynamicsymbols.
 
         """
-        name = 'generalized coordinate' if label == 'q' else 'generalized speed'
 
         def create_symbol(number):
             if n_coords == 1:
                 return dynamicsymbols(f'{label}_{self.name}')
             return dynamicsymbols(f'{label}{number}_{self.name}')
 
+
```

Comment:
I think standard formatting is for there to only be one blank line above and below a closure (no preceding blank line if the closure directly follows a docstring). Is this also the case in SymPy? I think this could be removed.

Reply:
yeah, my mistake